composeApp/src/main/kotlin/io/github/couchtracker/tmdb/TmdbRating.kt

```diff
@@ -1,28 +1,45 @@
 package io.github.couchtracker.tmdb
 
 import app.moviebase.tmdb.model.TmdbRatingItem
+import io.github.couchtracker.utils.deferredConstant
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
 import java.text.NumberFormat
 
+private val RATING_FORMATTER = deferredConstant {
+    NumberFormat.getInstance().apply {
```

Comment:
This was wrong: it didn't account for the locale changing. How do we plan to fix this? Should this maybe be a `Flow<NumberFormat>` that depends on the system languages? (after the other PR is merged)